docs/LICENSE_THIRD_PARTY

```diff
@@ -1,70 +0,0 @@
-######################################################################
```

Comment:
Need to update these files to remove this one.
https://github.com/search?q=repo%3Acupy%2Fcupy%20LICENSE_THIRD_PARTY&type=code